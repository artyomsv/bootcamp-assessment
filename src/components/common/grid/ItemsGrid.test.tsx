import * as React from 'react';
import {createShallow} from '@material-ui/core/test-utils';
import ItemsGrid from './ItemsGrid.widget';

const shallow = createShallow();

describe('renders without crashing', () => {
  test('if renders', () => {
    const mockFn = jest.fn();
    const result = shallow(<ItemsGrid values={[]} navigateTo={mockFn}/>);
    expect(result).toMatchSnapshot();
  });
});
