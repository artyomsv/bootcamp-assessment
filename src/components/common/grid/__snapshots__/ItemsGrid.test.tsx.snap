// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing if renders 1`] = `
<ItemsGrid
  classes={
    Object {
      "icon": "ItemsGrid-icon-2",
      "image": "ItemsGrid-image-6",
      "root": "ItemsGrid-root-1",
      "tile": "ItemsGrid-tile-3",
      "title": "ItemsGrid-title-4",
      "titleBar": "ItemsGrid-titleBar-7",
      "titleMovie": "ItemsGrid-titleMovie-5",
    }
  }
  navigateTo={[MockFunction]}
  values={Array []}
/>
`;
